### ISSUE TITLE
Remove inactive users

### ISSUE THREAD

**Issue by guimard** [linagora/twake-workplace#244]
Following #232 and CoSoft discussion, inactive users :
* will receive an SMS after 1 year of inactivity _(and maybe an email on recovery address)_ explaining that the account will be removed under 1 month
* will be deleted after 13 months of inactivity, then logout process should launch triggers:
  * [ ] to ToM server to remove all devices from Synapse database
  * [ ] to Tdrive ?
  * [ ] to James ?

**Comment by shepilov** [comment]
yes, probably also we will need to remove drive account and remove all the files from the storage

**Comment by guimard** [comment]
> yes, probably also we will need to remove drive account and remove all the files from the storage

Thanks, please insert a link to your API description when ready
